app/app.go

```diff
@@ -49,9 +49,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/gov"
```

Comment:
do we need to add / update the kava v20 migration handler to delete the mint module?